docs/topics/compatibility-guide-15.md

```diff
@@ -308,4 +308,28 @@ perspective
 > **Deprecation cycle**:
 >
 > - 1.4: deprecate `Double.toShort()/toByte()` and `Float.toShort()/toByte()` and propose replacement
-> - 1.5.0: raise the deprecation level to error
\ No newline at end of file
+> - 1.5.0: raise the deprecation level to error
+
```

Comment:
Is an empty line obligatory?

Reply:
Why do you ask? I don't know if it's technically required but it's always good to have one before each heading.